src/Client/Response/Base/Response.php

```diff
@@ -44,10 +44,8 @@ class Response
     /**
      * Original curl response in json format.
      * Original, unmodified response from curl request.
-     *
-     * @var mixed
      */
-    private $curlResponse;
+    private mixed $curlResponse;
```

Comment:
I think the issue is related to this change. Revert it and try again please. But also merge latest master please.